src/main/java/fi/fmi/avi/converter/ConversionHints.java

```diff
@@ -247,6 +247,16 @@
      */
     public static final Object VALUE_TAF_REFERENCE_POLICY_USE_OWN_VALID_TIME_ONLY = "USE_OWN_VALID_TIME_ONLY";
 
+    /**
+     * Controls how whitespace characters are handled when serializing TAC bulletin messages.
+     */
+    public static final Key KEY_WHITE_SPACE_PASSTHROUGH;
```

Comment:
How about if this conversion hint would specify the whitespace handling _mode_, with values for passthrough (instead of `VALUE_WHITE_SPACE_PASSTHROUGH_ENABLE`) and reformat/trim (former / default behavior)?

I would find it more specific as enabling passthrough will disable reformatting/trimming.